stanc.opam

```diff
@@ -3,7 +3,7 @@ opam-version: "2.0"
 synopsis: "The Stan compiler and utilities"
 depends: [
   "ocaml" {= "4.12.0"}
-  "dune" {>= "2.8"}
+  "dune" {>= "2.7" & >= "2.8"}
```

Comment:
What does this do? Looks kind of redundant?

Reply:
This was generated by dune because we require 2.8 but I had `lang-dune` set to 2.7, and I guess it wasn't smart enough to figure out that only one of those is needed. I'll fix it